gadget/update.go

```diff
@@ -237,7 +237,10 @@ func defaultPolicy(from, to *LaidOutStructure) bool {
 }
 
 // RemodelUpdatePolicy implements the update policy of a remodel scenario.
-func RemodelUpdatePolicy(_, _ *LaidOutStructure) bool {
+func RemodelUpdatePolicy(from, _ *LaidOutStructure) bool {
+	if !from.IsPartition() {
```

Comment:
I can actually think of a use case where this could make sense, eg. when updating a bootloader to a newer/different one. Perhaps we should only limit it to skip MBR role structures?